tests/test_middleware.py

```diff
@@ -1,11 +1,31 @@
 import pytest
+from django.http import HttpResponse
+from django.test import RequestFactory
```

Comment:
Here you don't need to import `RequestFactory`.

You can remove from `conftest.py` the following imports too:
```python
from django.contrib.auth.models import AnonymousUser
from django.contrib.sessions.middleware import SessionMiddleware
from django.test.client import RequestFactory
```

Reply:
fixed in 7613e57